__tests__/example.test.ts

```diff
@@ -23,4 +23,11 @@ test('Sync Response fails', async () => {
   const exampleSyncResFail = require('./example.sync.response.fail.json');
   const syncErrors = testlib.validate(exampleSyncResFail, 'sync');
   expect(syncErrors).toBeFalsy();
+});
+
+test('execute Response using actions an intent handler', async () =>{
+  const reqExec = testreq.generateExecuteRequest();
+  const res = fakeapp.onExecute(reqExec)
+  const testlibValid = testlib.validate(res, 'sync');
```

Comment:
`'execute'`?

Reply:
I'm going to remove this test in this PR for now and put it on a different branch and focus on sync for this one